backend/src/main/java/de/neuefische/backend/controller/UserController.java

```diff
@@ -0,0 +1,41 @@
+package de.neuefische.backend.controller;
+
+import de.neuefische.backend.service.MongoUserDetailsService;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.security.Principal;
+import java.util.Collection;
+
+
+@RestController
+@RequestMapping("/api")
+@CrossOrigin
```

Comment:
"proxy" in package.json?
läuft vermutlich besser als @crossorigin

Reply:
alles klar, wird gemacht.